### PR TITLE
Registration/Login - Authentication Tests

### PR DESCRIPTION
**Pull request by iPiyushPatel**
Work Item : https://github.com/etenlab/docs/issues/241

### CODE REVIEW

File: tests/data-factory/LoginData.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+class LoginData {
+  validLoginData() {
+    const logindata = Login;
+    logindata.email = 'AutomationUser@mailinator.com';
+    logindata.password = 'automation@123';
+    return logindata;
+  }
+}
+
+export default new LoginData();
```
Comment (lucky-gru) [inline tests/data-factory/LoginData.ts:12]: Add this configuration to the eslint.json file
```
    "import/no-anonymous-default-export": ["error", {
      "allowArray": false,
      "allowArrowFunction": false,
      "allowAnonymousClass": false,
      "allowAnonymousFunction": false,
      "allowCallExpression": true, // The true value here is for backward compatibility
      "allowNew": false,
      "allowLiteral": false,
      "allowObject": false
    }],
    ```
    
    And refactor the codebase without anonymous default export

File: tests/pages/Registration.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+  async isHeaderTextPresent() {
+    await this.page.locator(headerText).waitFor();
+    const headerTextPresnt = await this.page.locator(headerText).isVisible();
+    return headerTextPresnt;
+  }
+
+  async clickOnForgotPasswordButton() {
+    await this.page.locator(forgotPasswordButton).click();
+  }
```
Comment (lucky-gru) [inline tests/pages/Registration.ts]: fill empty line

**Review by lucky-gru — CHANGES_REQUESTED** [review]

File: tests/pages/Registration.ts (hunk 2 above)
Comment (iPiyushPatel) [reply tests/pages/Registration.ts]: Hello @lucky-gru 
Well, empty line is for differentiating two methods and it has been everywhere like that. Do you mean that empty line? Please advice.

File: tests/data-factory/LoginData.ts (hunk 1 above)
Comment (iPiyushPatel) [reply tests/data-factory/LoginData.ts:12]: Hello @lucky-gru 
Well, I couldn't find  eslint.json file , but I tired to put changes in .eslintrc.js it solves problem at code level but when I pushed code it gives me below error :

image
image



Am I missing anything ?

**Pull request closed without merging**